readme.md

```diff
@@ -16,19 +16,36 @@ Will run the audits defined in the `audits/` folder against a demo app. Chrome p
 # Start Chrome with a few flags
 ./launch-chrome.sh
 
-# Kick off a lighthouse run
+# set up the global
+npm link
 
-npm start
+# Kick off a lighthouse run
+lighthouse
+lighthouse https://airhorner.com/
 ```
 
+## Chrome Extension
+
+See [./extension](https://github.com/GoogleChrome/lighthouse/tree/master/extension).
+
 ## Tests
 
-Some basic unit tests forked from node-big-rig are in `/test` and run via mocha. eslint is also checked for style violations.
+Some basic unit tests forked are in `/test` and run via mocha. eslint is also checked for style violations.
 
 ```js
 npm test
 ```
 
+### Code Style
```

Comment:
FYI: I started some readme notes on code style. Let's try to document our decisions there.